README.md

```diff
@@ -0,0 +1,16 @@
+Trailing whitespaces checker
+============================
+
+This little bash script recursively iterates all files in the current directory and reports those that have lines with trailing whitespaces. In case a file is found, the script will exit with status code `3`, or `0` otherwise.
```

Comment:
out of curiosity, why 3 ?

Reply:
Because 1 and 2 are reserved for some specific cases: http://www.tldp.org/LDP/abs/html/exitcodes.html.